mistralrs-core/src/pipeline/mod.rs

```diff
@@ -1,5 +1,5 @@
 mod cache_manager;
-mod chat_template;
+pub mod chat_template;
```

Comment:
Not sure if this module should be made public, but seems like might have been required for the docs reference to it?

Reply:
Yeah, makes sense. I actually want to refactor the macros to remove the `#[macro_export]` usage later.